src/components/header/NavBarDefault.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import DropDown from '../inAppNotification/notification/dropDown';
+// import AppNotificatonContainer from '../inAppNotification/AppNotificatonContainer';
```

Comment:
Nice job on this PR. Please take out the commented code if you no longer need them

Reply:
Thanks for your feedback. I will remove all commented codes now